html-node-core/src/node/doctype.rs

```diff
@@ -0,0 +1,37 @@
+use std::fmt::{self, Display, Formatter};
+
+/// A doctype.
+///
+/// ```html
+/// <!DOCTYPE html>
+/// ```
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Doctype {
+    /// The value of the doctype.
+    ///
+    /// ```html
+    /// <!DOCTYPE synax>
```

Comment:
(this is my typo) but it would be appreciated if you could fix this to `syntax` because i don't wanna create merge conflicts :)

Reply:
fixed